### PR TITLE
SoftwareTeam.ReleasePolicy

### PR DESCRIPTION
**Pull request by matyasselmeci**
https://twiki.opensciencegrid.org/bin/view/SoftwareTeam/ReleasePolicy

- [ ] Update spreadsheet w/ merge date
- [ ] Replace twiki page w/ header

### CODE REVIEW

**Comment by matyasselmeci** [comment]
@timtheisen, could you please look this over and see if it's still useful, or is there a more up-to-date description of our release policy elsewhere?

**Comment by timtheisen** [comment]
This is the best that we have for now. We need to update it sometime.

File: docs/policy/release-policy.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+Software Release Policy
+=======================
+
+This document doesn't talk about technical details of how to do a release. [The release process is discussed elsewhere](../release/cut-sw-release/).
+
+1.0 Software Repositories
```
Comment (brianhlin) [inline docs/policy/release-policy.md]: I'm not a fan of the numbered headers, could we get rid of them?

**Comment by matyasselmeci** [comment]
Fixed.

**Pull request merged**